pulpcore/app/viewsets/custom_filters.py

```diff
@@ -277,8 +277,12 @@ class CharInFilter(BaseInFilter, CharFilter):
 class LabelFilter(Filter):
     """Filter to get resources that match a label filter string."""
 
+    label_field_name = "pulp_labels"
+
     def __init__(self, *args, **kwargs):
         kwargs.setdefault("help_text", _("Filter labels by search string"))
+        if "label_field_name" in kwargs:
+            self.label_field_name = kwargs.pop("label_field_name")
```

Comment:
```suggestion
        if "label_field_name" in kwargs:
            self.label_field_name = kwargs.pop("label_field_name")
        else:
            self.label_field_name = "pulp_labels"
```
Let's not mix class and instance variables.